main.go

```diff
@@ -66,13 +69,13 @@ func saveResults(out chan DnsRecordResult) {
 		panic(err)
 	}
 
-	ins, err := db.Prepare("INSERT INTO dns_records (`domain`, `type`, `value`) VALUES (?, ?, ?)")
+	ins, err := db.Prepare("INSERT INTO dns_records (`domain`, `type`, `value`, `ip_dec`) VALUES (?, ?, ?, ?)")
 	if err != nil {
 		panic(err)
 	}
 
 	for result := range out {
-		ins.Bind(result.Domain, dns.TypeToString[result.RecordType], result.Value)
+		ins.Bind(result.Domain, result.RecordType, result.Value, result.IPDec.String())
```

Comment:
Actually we have on Line `102` the following:

```answerRR := dns.TypeToString[record.Header().Rrtype]
recordString := getRecordString(answerRR, record)```

and later in the struct

```
RecordType: answerRR,
```

Meaning we are passing the `dns.TypeToString` string into the `map` and using it like this globally.

Side note, We (from this pull request) are storing the RR Type that is returned rather than the one asked. _Technically_, there is a possible we ask for `A` record and instead get back a `CNAME`